.docker/images/govcms8/scripts/govcms-deploy

```diff
@@ -41,19 +50,21 @@ common_deploy () {
 
 # Non production environments.
 if [[ "$LAGOON_ENVIRONMENT_TYPE" != "production" ]]; then
+
+    read_replica_enabled $sync_opts
+
     if ! drush status --fields=bootstrap | grep -q "Successful"; then
         # Import prod db in Lagoon development environments.
         if [[ ! -z "$LAGOON_ENVIRONMENT_TYPE" && "$LAGOON_ENVIRONMENT_TYPE" != "local" ]]; then
-          drush sql-sync @govcms.prod @self -y
-          common_deploy
+          drush sql-sync @govcms-prod @self $sync_opts -y
```

Comment:
```suggestion
          drush sql-sync @govcms-prod @self $sync_opts -y
          common_deploy
```

Reply:
Think you accidentally removed common_deploy -- still want updb+cim to run after syncing from r/o db